docs/Reference/CLI/Subcommands/Voluntary-Exit.md

```diff
@@ -59,6 +59,39 @@ Endpoint of the beacon node's REST API. Default is `http://127.0.0.1:5051`.
 The path to the YAML configuration file.
 The default is `none`.
 
+### confirmation-enabled
+
+=== "Syntax"
+
+    ```bash
+    teku voluntary-exit --confirmation-enabled=<BOOLEAN>
+    ```
```

Comment:
can we make like a big bold warning that says that confirmation-enabled=false is a really really bad idea, and if you get it wrong it will exit validators very quickly?
Basically by default it will ask for confirmation that the passed validators be exited, and passing that flag will not ask, it will just start sending exits. helpful for testing and docker, but dangerous...